src/main/resources/static/js/courseform.js

```diff
@@ -3,6 +3,17 @@ $(document).ready(function() {
         $('#courseForm').slideToggle();
     });
 
+    $("#course-form-submit").click(function() {
+
+        if(document.forms["courseForm"]["section"] == "") {
+            document.forms["courseForm"]["section"] = null;
+        }
+
+        if(document.forms["courseForm"]["startDate"].value == ""){
+            document.forms["courseForm"]["startDate"].value = "1970-01-01";
+        }
+    });
+
```

Comment:
These are kind of an ugly solution, but empty fields get parsed as empty strings when they are sent to the server which produces a 400 error for these fields. If we send some garbage data however, we can get around this. Does anyone know a less hacky solution for this?

Reply:
I'm not sure what you mean by this, do you mean you get a 400 if you try to submit with an empty date?

Also, it doesn't really matter but we've been mostly sticking to jquery for stuff like those selectors... although the pure js is nicer. This should be the jquery equivalent (I believe): ` $('input#fieldName').val();`

I'm really torn on whether or not you should change this